content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Views'
+metaTitle: 'How to include views in your Prisma schema'
+metaDescription: 'How to include views in your Prisma schema'
+hidePage: false
+preview: true
+tocDepth: 2
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+Support for views is currently a very early, experimental preview feature.
+
+</Admonition>
+
+<!-- TODO: add more to admonition -->
+
+Database views allow you to name and store a query. In relational databases, views are stored queries that may include columns in multiple tables. In MongoDB, views are queryable objects where the contents are defined by an [aggregation pipeline](https://www.mongodb.com/docs/manual/core/aggregation-pipeline) on other collections.
+
+This guide explains how to enable the `views` preview feature and add views to your Prisma schema.
+
+</TopBlock>
+
+## How to enable the <inlineCode>views</inlineCode> preview feature
+
+Support for views is currently in an early preview. To enable the `views` preview feature, add the `views` feature flag to the `previewFeatures` field of the `generator` block in your Prisma schema file:
+
+```prisma file=schema.prisma highlight=3;add
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["views"]
+}
+```
+
+## How to use views in your Prisma schema
+
+To add a view to your Prisma schema, use the `view` keyword. For example, take the following Prisma schema with a `User` model with a linked `Profile`:
+
+```prisma file=schema.prisma
+model User {
+  id      Int      @id @default(autoincrement())
+  email   String   @unique
+  name    String?
+  profile Profile?
+}
+
+model Profile {
+  id     Int    @id @default(autoincrement())
+  bio    String
+  user   User   @relation(fields: [userId], references: [id])
+  userId Int    @unique
+}
+```
+
+The following `UserInfo` view combines the `email` and `name` fields from the `User` model and the `bio` field from the `Profile` model:
```

Comment:
Technically the `view` itself does not combine the fields of the other models, but the underlying SQL does. How it is worded right now could imply that jsut adding a view like this does anything - but it does not. This is just for illustration.

Reply:
I've rewritten and moved things around a bit, hopefully better?